File: Latihan_B/php/Processor.php
<?php
class Processor {
    private $name;
    private $price;

    // constructor
    function __construct($name, $price) {
        $this->name = $name;
        $this->price = $price;
    }

    // method set untuk name dan price
    function setName($name) {
        $this->name = $name;
    }
    function setPrice($price) {
        $this->price = $price;
    }
    // method get untuk name dan price
    function getName() {
        return $this->name;
    }
    function getPrice() {
        return $this->price;
    }

    // method untuk output Processor
    function printProcie() {
        echo "Processor". "<br/>";
        echo "-------------". "<br/>";
        echo "Name        : ".  $this->getName(). "<br/>";
        echo "Price       : ". "Rp. ". $this->getPrice(). "<br/>";
    }

    // destructor
    function __destruct() {}
}
?>

File: Latihan_B/java/Processor.java
public class Processor 
{
    private String name;
    private int price;

    // constructor
    public Processor(){}
    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    // method set untuk name dan price
    public void setName(String name) {
        this.name = name;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    // method get untuk name dan price
    public String getName() {
        return name;
    }
    public int getPrice() {
        return price;
    }

    // method output untuk Processor
    public void printProcie() {
        System.out.println("Processor");
        System.out.println("-------------");
        System.out.println("Name        : " + name);
        System.out.println("Price       : " + "Rp. " + price);
    }
}
